### ISSUE TITLE
Add Imagemagick package to install script

### ISSUE THREAD

**Issue by lwo** [IQSS/dataverse#3035]
ImageMagick is invoked in the ImageThumbConverter class and its location can be set here:

https://github.com/IQSS/dataverse/blob/develop/doc/sphinx-guides/source/installation/config.rst#dataverse-path-imagemagick-convert

Should ImageMagick become part of the installation procedure? Should it be mentioned in the prerequisites.rst docs or is its use optional?

**Comment by pdurbin** [comment]
> Should ImageMagick become part of the installation procedure?

Yes, probably, but per #8014 we might want to support GraphicsMagick rather than ImageMagick. dataverse-ansible made the switch in https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible/pull/110

> Should it be mentioned in the prerequisites.rst docs or is its use optional?

Yes, this was done a while back in 9e604cb as part of pull request #3459. Here's how it looks at https://guides.dataverse.org/en/5.8/installation/prerequisites.html#imagemagick

![Screen Shot 2021-11-29 at 3 32 40 PM](https://user-images.githubusercontent.com/21006/143938790-be7071ae-1071-493a-8d8b-cf4b17ced2e3.png)

My only comment here is that we might want to move the macOS line to the dev guide since we tend to keep the installation guide focused on production installations, which run Linux.

I'm here because my dev environment doesn't currently create thumbnails for PDFs as I work on #8160. I guess I'll try GraphicsMagick on my Mac if I can.

**Comment by qqmyers** [comment]
FWIW - pdfs can be a special case - https://github.com/IQSS/dataverse/issues/5617

**Comment by pdurbin** [comment]
> I'm here because my dev environment doesn't currently create thumbnails for PDFs as I work on #8160. I guess I'll try GraphicsMagick on my Mac if I can.

As part of PR #8260 I ended up running (and documenting in the dev guide) `brew install imagemagick`: https://guides.dataverse.org/en/5.12/developers/tips.html#thumbnails

For developers, think this is ok.

As I mentioned above, between 2016 when this issue was opened and now we did start indicating on the "prereq" page that you have to install ImageMagick.

I think that's enough to consider this issue done. There's a heads up in the docs to both sysadmins and devs that ImageMagick is required. I don't think we need to add this to the installer.

@lwo you opened the issue. What do you think? 😄

**Comment by lwo** [comment]
I agree.
